opengrok-indexer/src/main/java/org/opengrok/indexer/index/Indexer.java

```diff
@@ -1147,4 +1147,8 @@ private static void pauseToAwaitProfiler() {
 
     private Indexer() {
     }
+
+    public static OptionParser getOptParser() {
```

Comment:
I wouldn't add getter just because test needs it. If test needs the access, then I would suggest using reflection.

Reply:
+1

Reply:
Any hints on how to do it ? I tried the method described in https://stackoverflow.com/questions/1196192/how-to-read-the-value-of-a-private-field-from-a-different-class-in-java however Indexer is static, its constructor is private and getClass() cannot be called from static context (at least IDEA says so).

Reply:
You do not have to use `getClass()` method. You can use class field -> `Indexer.class`.